Assignment_1.cu

```diff
@@ -0,0 +1,45 @@
+#include<iostream>
+using namespace std;
+int Array_Size;
+
+__global__ void Sum(float* d_in1,float* d_in2, float* d_out,int* d_array_size)
+{
+    int id = blockIdx.x * blockDim.x + threadIdx.x;
+    if(id < *d_array_size)
+     d_out[i] = d1_in[i] + d2_in[i];    
+}
+int main()
+{
+    cout << "Enter the array size : ";
+    cin >> Array_Size;
+	
+    float h_in1[Array_Size],h_in2[Array_Size] h_out[Array_Size];
```

Comment:
float h_in1[Array_Size],h_in2[Array_Size], h_out[Array_Size]; // You forgot the comma after h_in2.